src/pages/install/linux.md

```diff
@@ -0,0 +1,31 @@
+
+<hgroup>
+
+← [Back to install instructions](/install)
+
+# Installing RGBDS on Linux
+
+</hgroup>
+
+First, please check if RGBDS is available in your distribution's software center or package manager repositories.
+If a satisfactory version is available, it is a good idea to install from there.
+
+If not, and you want to install version v0.6.1 or newer, you can download our pre-built binaries.
+
+1. Click the link of the version you want [from the list](/docs).
+2. Under the "GitHub links" heading, follow the "release page" link.
```

Comment:
Should we standardize on some specific verb for links? (If "click" is too mouse-centric, "follow" could instead work for all of them.)
```suggestion
2. Under the "GitHub links" heading, click the "release page" link.
```

Reply:
I prefer 'follow' in those occurences